include/tvm/tir/transform.h

```diff
@@ -29,6 +29,14 @@
 #include <tvm/tir/function.h>
 
 #include <string>
+#include <vector>
+
+// Forward declarations from tvm::relay
+namespace tvm {
+namespace relay {
+class ConstantNode;
+}  // namespace relay
+}  // namespace tvm
```

Comment:
We probably don't need these lines, right?

Reply:
We don't. Removed.